packages/actor-rdf-resolve-quad-pattern-federated/test/__mocks__/MediatorRdfMetadataAggregateMock.ts

```diff
@@ -0,0 +1,55 @@
+import {Record} from "immutable";
+
+export const mockedMediatorAgg = {
```

Comment:
Not need to make this a separate file, just add it in the test file.

Also, we can probably make a much more simplified version of this mediator instead of copy-pasting the existing totalItems actor.

Reply:
The mocked RDF Metadata Aggregate mediator (`mockedMediatorAgg`) is used by both `FederatedQuadSource-test.ts` and `ActorRdfResolveQuadPatternFederated-test.ts`, so that would imply duplicating that code twice?